core/API/DataTableManipulator/Flattener.php

```diff
@@ -38,17 +38,16 @@ public function includeAggregateRows()
      * Separator for building recursive labels (or paths)
      * @var string
      */
-    public $recursiveLabelSeparator = ' - ';
+    public $recursiveLabelSeparator = '';
 
     /**
      * @param  DataTable $dataTable
+     * @param string $recursiveLabelSeparator
      * @return DataTable|DataTable\Map
      */
-    public function flatten($dataTable)
+    public function flatten($dataTable, $recursiveLabelSeparator)
```

Comment:
$recursiveLabelSeparator is no longer hard coded / hacked in the Flattener, it can be defined by each report now.